packages/rest-crud/src/repository-builder.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/rest-crud
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {
+  DefaultCrudRepository,
+  Entity,
+  EntityCrudRepository,
+  juggler,
+} from '@loopback/repository';
+import * as assert from 'assert';
+
+export function defineCrudRepositoryClass<
```

Comment:
Missing API docs, please add.

Reply:
I am afraid this is not resolved yet. By API docs I mean a `tsdoc` comment, see what we have for the controller factory.